test/ha/controller_test.go

```diff
@@ -32,7 +32,10 @@ import (
 	"knative.dev/serving/test/e2e"
 )
 
-const controllerDeploymentName = "controller"
+const (
+	controllerDeploymentName = "controller"
+	numReconcilers           = 7 // Keep in sync with ./cmd/controller/main.go
```

Comment:
a common place of sorts, perhaps?

Reply:
I added main_test.go in cmd/controller to ensure the constant stays in sync.